main_logic/tts_client.py

```
@@ -1030,3 +1044,175 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
     else:
         logger.error(f"{core_api_type}不支持原生TTS，请使用自定义语音")
         return dummy_tts_worker
+
+
+def local_cosyvoice_worker(request_queue, response_queue, audio_api_key, voice_id):
+    """
+    本地 CosyVoice WebSocket Worker
+    适配 model_server.py 定义的 /api/v1/ws/cosyvoice 接口
+    """
+    import uuid # 确保导入uuid
```

Ruff (0.14.10)

Redefinition of unused `uuid` from line 16

Remove definition: `uuid`

(F811)